packages/patternfly-4/react-core/src/components/DataList/DataListAction.js

```diff
@@ -28,14 +46,22 @@ class DataListAction extends React.Component {
       id,
       'aria-label': ariaLabel,
       'aria-labelledby': ariaLabelledBy,
+      hiddenBreakpoint,
+      visibleBreakpoint,
       ...props
     } = this.props;
 
     return (
-      <div className={css(styles.dataListItemAction, className)}>
-        <div className={css(styles.dataListAction, className)} {...props}>
-          {children}
-        </div>
+      <div
+        className={css(
+          styles.dataListItemAction,
+          breakPointTable.visible[visibleBreakpoint],
```

Comment:
Let's write some utility classes to do what @mcoker suggests here that are reusable for other components. We seem to be lacking it.

Reply:
absolutely!